Src/AutoFakeItEasy/FakeItEasyType.cs

```diff
@@ -30,5 +30,10 @@ internal static Type GetFakedType(this Type type)
         {
             return type.GetTypeInfo().GetGenericArguments().Single();
         }
+
+        internal static bool IsDelegate(this Type type)
+        {
+            return type != null && type.GetTypeInfo().IsSubclassOf(typeof(Delegate));
```

Comment:
See the comment [here](https://github.com/AutoFixture/AutoFixture/blob/9121cedad8b8eddc8993552ba87a06fbc92f97be/Src/AutoMoq/MockType.cs#L52). Should we use `MulticastDelegate` instead?

Reply:
Will do.